docs/backup_restore_guide.md

```diff
@@ -0,0 +1,69 @@
+# Harbor Backup and Restore Guide
+
+When the container registry storage is local file system, the straightforward way to backup a standalone Harbor data is to backup the /data directory and recover it to /data directory.
```

Comment:
what happens with in-flight data in the /data directory? is this a safe way to backup where no corruption can occur?

Reply:
it is not. to avoid data corruption it is necessary to ensure that there is no activity on the platform - remember that Harbor uses PostgreSQL. to have a useful backup of Harbor, at least we would need to use `pg_dump` or similar tool for the database.